sycl/source/detail/graph_impl.cpp

```diff
@@ -226,6 +228,133 @@ bool graph_impl::clearQueues() {
   return AnyQueuesCleared;
 }
 
+/// Visits a node on the graph and it's successors recursively in a depth-first
+/// approach.
+/// @param Node The current node being visited.
```

Comment:
if we could put `[in],[out], [in,out]` on these param descriptions, it would be helpful for knowing how the references are used.

Reply:
Done!